### DISCUSSION TITLE
Is there a way include assets/media from a module into the main layout outside of OrchardCore CMS?

### DISCUSSION THREAD

**Discussion by MikeAlhayek** [OrchardCMS/OrchardCore#7673] [Q&A] [answered]
I am trying to use OrchardCore in a project where I need multi-tenancy and module platform. I am not going to be using the CMS functionality here. However, I need to find a way for my modules to include media/assets into the main layout outside the CMS. For example, one module will compose additional CSS/JS code that needs to be included in the layout. How can I do that with OrchardCore outside of the OrchardCoreCMS?

Thank for your help

**Comment by deanmarcussen** [comment, marked as answer]
The `OrchardCore.ResourceManagement` project is not bound to the CMS, and intended for use such as you describe in a framework project.

It includes the tag helpers that you will find useful to add resources in a modular way

**Comment by MikeAlhayek** [comment]
@deanmarcussen is there a list of all packages that can be used outside of the CMS?
